### DISCUSSION TITLE
student devloper pack

### DISCUSSION THREAD

**Discussion by TirthShah007** [community/community#86056] [GitHub Education]
### Hi everyone,

i can able to send my student devloper pack details..i upload document in proper resolution and storage.....when i press Process my application...it's show processing information..but can be send successfully till...it's still show processing information and also doesn't show any error

**Comment by OtmaneDaoudi** [comment]
Having the same issue!
I tried changing the browser, changing the device, and removing adblockers, but nothing seemed to work.
After inspecting the page, i found an error that says : Uncaught TypeError: Cannot read properties of undefined (reading 'redirect_url')

**Comment by juanpflores** [comment]
Hello @TirthShah007,

I am pleased to inform you that your academic status has been verified successfully. Congratulations on this achievement!

You can now access [GitHub Global Campus](https://education.github.com/globalcampus/student), which provides you with an opportunity to connect with local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, and discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more.

You may have already received an email notification regarding your academic benefits. If not, please wait for up to 72 hours for your benefits to be available.

You can always check your current academic status by clicking [here](https://education.github.com/discount_requests/application).
